addons/rose/addon/components/rose/button/index.hbs

```diff
@@ -5,7 +5,7 @@
 
 <button
   ...attributes
-  type={{if @submit 'submit' 'button'}}
+  type={{if @onSubmit 'submit' 'button'}}
```

Comment:
Why are we changing the argument to `onSubmit`?

Reply:
@DhariniJeeva `no-passed-in-event-handlers` lint error from `ember-template-lint`